### ISSUE TITLE
Install Error - Temp File Already Exists

### ISSUE THREAD

**Issue by gpmidi** [Khroki/MCEdit-Unified#253]
If I stop an install and then rerun it I get the error below. 
```
Start step 2 (dependencies installation) [y/n]? y
Traceback (most recent call last):
  File "../bin3/mcedit/install.py", line 468, in <module>
  File "../bin3/mcedit/install.py", line 381, in IIIIii1I
OSError: [Errno 17] File exists: 'temp'
```

**Comment by gpmidi** [comment]
The error was when trying to install mcedit on a Ubuntu 14.04 based system. It was run as root from /home/gpmidi/Downloads. More output can be found here: https://www.dropbox.com/s/rt29311vxjo6ock/console.txt?dl=0

**Comment by LaChal** [comment]
As a workaround, delete the 'mcedi' folder created by the installer before rerun it.
This will be fixed in the next release.

By the way, you don't need to be logged as root (or use a root term) to install MCEdit.
Root privileges are needed to install binary dependencies (outside the installer).
You will be prompted for a root password if you decide to install the Python dependencies system wide.

**Comment by LaChal** [comment]
Fixed in release 1.2.5.0.
